<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QGetWidget</class>
 <widget class="QWidget" name="QGetWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1436</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QGetWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#widget_main
{
border-image: url(:/Image/Resources/Image/img_bg.png);
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_main" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_title" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>95</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>95</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>25</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>25</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="btn_back">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">#btn_back
{
min-width:55px;
max-width:55px;
min-height:50px;
max-height:50px;
border-radius: 10px;
}
#btn_back{border-image: url(:/Image/Resources/Image/btn_Return_nor.png);}
#btn_back:hover{border-image:url(:/Image/Resources/Image/btn_Return_sele.png);}
#btn_back:pressed{border-image:url(:/Image/Resources/Image/btn_Return_press.png);}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_title">
           <property name="minimumSize">
            <size>
             <width>504</width>
             <height>71</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>504</width>
             <height>71</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">
min-width:504px;
max-width:504px;
min-height:71px;
max-height:71px;
</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btn_home">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">#btn_home
{
min-width:55px;
max-width:55px;
min-height:50px;
max-height:50px;
border-radius: 10px;
}
#btn_home{border-image: url(:/Image/Resources/Image/btn_home_nor.png);}
#btn_home:hover{border-image:url(:/Image/Resources/Image/btn_home_sele.png);}
#btn_home:pressed{border-image:url(:/Image/Resources/Image/btn_home_press.png);}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0,0,0,1">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QWidget" name="widget_get" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>80</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QToolButton" name="btn_QRGet">
             <property name="minimumSize">
              <size>
               <width>312</width>
               <height>143</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>312</width>
               <height>143</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>36</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">#btn_QRGet
{
min-width:312;
max-width:312;
min-height:143;
max-height:143px;
border-radius: 10px;
}
#btn_QRGet{border-image: url(:/Image/Resources/Image/btn_qujian1_nor.png);}
#btn_QRGet:hover{border-image:url(:/Image/Resources/Image/btn_qujian1_sele.png);}
#btn_QRGet:pressed{border-image:url(:/Image/Resources/Image/btn_qujian1_press.png);}</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="btn_NumGet">
             <property name="minimumSize">
              <size>
               <width>312</width>
               <height>143</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>312</width>
               <height>143</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>36</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">#btn_NumGet
{
min-width:312;
max-width:312;
min-height:143;
max-height:143px;
border-radius: 10px;
}
#btn_NumGet{border-image: url(:/Image/Resources/Image/btn_qujian2_nor.png);}
#btn_NumGet:hover{border-image:url(:/Image/Resources/Image/btn_qujian2_sele.png);}
#btn_NumGet:pressed{border-image:url(:/Image/Resources/Image/btn_qujian2_press.png);}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="popupMode">
              <enum>QToolButton::DelayedPopup</enum>
             </property>
             <property name="autoRaise">
              <bool>false</bool>
             </property>
             <property name="arrowType">
              <enum>Qt::NoArrow</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>6</number>
          </property>
         </layout>
        </item>
        <item>
         <widget class="QWidget" name="widget_Save" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <property name="spacing">
            <number>80</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QToolButton" name="btn_QRSave">
             <property name="minimumSize">
              <size>
               <width>312</width>
               <height>143</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>312</width>
               <height>143</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>36</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">#btn_QRSave
{
min-width:312;
max-width:312;
min-height:143;
max-height:143px;
border-radius: 10px;
}
#btn_QRSave{border-image: url(:/Image/Resources/Image/btn_cunjian1_nor.png);}
#btn_QRSave:hover{border-image:url(:/Image/Resources/Image/btn_cunjian1_sele.png);}
#btn_QRSave:pressed{border-image:url(:/Image/Resources/Image/btn_cunjian1_press.png);}</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="btn_NumSave">
             <property name="minimumSize">
              <size>
               <width>312</width>
               <height>143</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>312</width>
               <height>143</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>36</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">#btn_NumSave
{
min-width:312;
max-width:312;
min-height:143;
max-height:143px;
border-radius: 10px;
}
#btn_NumSave{border-image: url(:/Image/Resources/Image/btn_cunjian2_nor.png);}
#btn_NumSave:hover{border-image:url(:/Image/Resources/Image/btn_cunjian2_sele.png);}
#btn_NumSave:pressed{border-image:url(:/Image/Resources/Image/btn_cunjian2_press.png);}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="popupMode">
              <enum>QToolButton::DelayedPopup</enum>
             </property>
             <property name="autoRaise">
              <bool>false</bool>
             </property>
             <property name="arrowType">
              <enum>Qt::NoArrow</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtControl.qrc"/>
 </resources>
 <connections/>
</ui>
